.github/workflows/deploy.yml

```
@@ -0,0 +1,58 @@
+          key: ${{ secrets.AWS_EC2_SSH_PRIVATE_KEY }}
+          script: |
+            docker pull ${{ secrets.DOCKERHUB_USERNAME }}/${{ github.event.repository.name }}:latest
+            docker stop ${{ github.event.repository.name }}-${{ vars.ENV }} || true
+            docker rm ${{ github.event.repository.name }}-${{ vars.ENV }} || true
+            docker run -d -p ${{ vars.PORT }}:8000 \
+              --name ${{ github.event.repository.name }}-${{ vars.ENV }} \
+              --network=${{ vars.DOCKER_NETWORK }} \
+              -e DB_NAME="${{ secrets.DB_NAME }}" \
+              -e MONGODB_URI="${{ secrets.MONGODB_URI }}" \
+              -e ALLOWED_HOSTS="${{ vars.ALLOWED_HOSTS }}" \
+              ${{ secrets.DOCKERHUB_USERNAME }}/${{ github.event.repository.name }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)